doc/nimc.rst

```diff
@@ -681,6 +681,15 @@ nimMemAlignTiny
 Sets `MemAlign` to `4` bytes which reduces the memory alignment
 to better match some embedded devices.
 
+Thread stack size 
+=================
+
+Nim's thread API provides a simple wrapper around more advanced
+RTOS task features. Customizing the stack size and stack guard size can
+be done by setting `-d:StackThreadSize=16384` or `-d:StackGuardSize:8`.
```

Comment:
These must be named `nimStackThreadSize` and `nimStackGuardSize`

Reply:
Renamed them, went with `nimThreadStackSize` and `nimThreadStackGuard` to be a bit clearer.